pkg/performanceprofile/controller/performanceprofile/components/utils.go

```
@@ -103,9 +116,11 @@ func NewCPULists(reservedList, isolatedList string) (*CPULists, error) {
 	if err != nil {
 		return nil, err
 	}
+	offlined, _ := cpuset.Parse(offlinedList)
```

why are we ignoring the error here?

---

This also triggers a bell. In a future PR, we should make sure we get and handle the parsing errors for the cpu sets above.

---

missed. fix.

---

It seems to me we are still ignoring the error? should be
```
offlined, err := cpuset.Parse(offlinedList)
if err != nil {
   ...
```